### PR TITLE
Adds False Armblade Sting / Nerfs Transformation Sting

### PR DESCRIPTION
**Pull request by Ikarrus**
Because transform stings are way too bullshit easy to make the station completely unable to function.

When I originally made transform stings work on living targets it was never meant to be a stealthy way to spread chaos. They were originally meant as a tool to frame others for your own actions. They have to carry at least some small element of risk because once it gets going it will never stop.

They will however, give the target a very brief stun and jitter. The victim is basically being forced to transform as well, so the process will be slightly delayed but painful. The cost will also increase to 3.

To compensate I'm adding the False Armblade Sting. This sting forces a target to grow a (useless) arm blade that lasts roughly a minute. This will send the same message as changelings do when their arm blades form. With this, changelings have a proper way to frame others.

As a bonus, all changeling mutations (including the fake one) now make the sickening crunch sound the message suggests.

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 1]
```diff
@@ -91,8 +91,17 @@
 	var/datum/dna/NewDNA = selected_dna
 	if(ismonkey(target))
 		user << "<span class='notice'>We stealthily sting [target.name].</span>"
```
Comment (Anonus) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: >stealthily

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 2]
```diff
@@ -91,8 +91,17 @@
...
 	if(ismonkey(target))
 		user << "<span class='notice'>We stealthily sting [target.name].</span>"
-	hardset_dna(target, NewDNA.uni_identity, NewDNA.struc_enzymes, NewDNA.real_name, NewDNA.blood_type, NewDNA.species.type, NewDNA.features)
-	updateappearance(target)
+	target.forcesay(hit_appends)
+	target.visible_message("<span class='danger'>[target] begins to violenty convulse!</span>","<span class='userdanger'>You feel a tiny prick and a begin to violently convulse!</span>")
+	if(iscarbon(target) && (target.status_flags & CANWEAKEN))
+		var/mob/living/carbon/C = target
+		C.Stun(3)
+		C.Weaken(3)
```
Comment (Anonus) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: I'll bet you money that this and mutesting get used together.

**Comment by Xhuis** [comment]
Looks good but I think the message the victim gets should be a bit differently worded, the way it is right now makes it seem like the convulsions are voluntary.

File: code/game/gamemodes/changeling/powers/tiny_prick.dm [hunk 3]
```diff
@@ -90,9 +90,19 @@
...
-	hardset_dna(target, NewDNA.uni_identity, NewDNA.struc_enzymes, NewDNA.real_name, NewDNA.blood_type, NewDNA.species.type, NewDNA.features)
-	updateappearance(target)
+		user << "<span class='notice'>We sting [target.name].</span>"
+
+	if(iscarbon(target) && (target.status_flags & CANWEAKEN))
+		var/mob/living/carbon/C = target
+		C.Stun(3)
+		C.Weaken(3)
+		C.do_jitter_animation(100)
+		C.take_organ_damage(20, 0) //The process is extremely painful
```
Comment (Cheridan) [inline code/game/gamemodes/changeling/powers/tiny_prick.dm]: for you

**Comment by freerealestate** [comment]
They already aren't stealthy though. They rely on finding a group to transform people in, otherwise it's really obvious to the person you transformed that you're a changeling.

**Comment by Fox-McCloud** [comment]
What? This is nuts. Hell no.

Adding a stun+weaken do it that lasts ~6 seconds lets the 'ling get no less than 7 armblade attacks in. This is going back to oldling where it was sting->stun->dead no counters. Not only that, but it does damage to boot; they'll be conked clean out in 4 hits, and there's not a thing they can do about it.

you've effective changed this from a gag/stealth/chaos tool into a full out and out combat tool.

:-1:  x :100: weren't there already huge complaints about how horrendously overpowered lings currently are?

**Comment by Ikarrus** [comment]
well okay I removed the stun

**Comment by Fox-McCloud** [comment]
Still iffy about the damage it gives, but overall, it's an expensive ability, anyway--soo...I'm just happy it doesn't have the crazy stun with it! 

:+1:

**Comment by Alek2ander** [comment]
:-1: on the jitter.
Removes the LOOK HE TRANSFORMED LYNCH THE LING misdirection element

**Comment by Ikarrus** [comment]
I don't think anyone really falls for that any more. 

It used to work when changeling transforms were super obvious to everyone and the sting gave the same message, but it's been since made a lot more stealthy and unnoticable. I wouldn't mind adding the message back though. 

As it is now the transform sting is a chaos tool. A pretty shitty one that fucks over other changelings that people have asked me to just remove outright, but hey.

**Comment by Ikarrus** [comment]
Besides the jutter only lasts a couple of seconds.

**Comment by freerealestate** [comment]
> A pretty shitty one that fucks over other changelings

It sounds like that issue should be solved then instead.

I still don't get how you consider transform sting stealthy when it
immediately transforms people and only people adjacent to you. You don't
often find situations (at least in low pop) where you have at least two
people standing adjacent to you so you have plausible deniability when you
transform someone. It's a very risky move that can out you as a changeling
immediately.

**Comment by Ikarrus** [comment]
>It sounds like that issue should be solved then instead.

I'd like to just outright remove transform stings, then.

**Comment by freerealestate** [comment]
You could just remove the disguised escape objective, I think less people
would care about that then the removal of this sting.

**Comment by Perakp** [comment]
I dislike the very existence of transform sting. It's almost a "do nothing" sting, like the extract DNA one, pretty much no one cares if your name doesn't match your ID card when the sting exists because you can just say you got stung, it doesn't create interesting gameplay, tension, drama, anything. I don't get the point of it, beyond "luls".

:+1: on the nerf.

**Comment by Ikarrus** [comment]
That's exactly how I feel, too, perakp

What i might do is go with my original plan and just replace the transform sting with an ability that lets you regurgitate a duplicate version of yourself.

**Comment by Miauw62** [comment]
remove ling

**Comment by Iamgoofball** [comment]
Keep transform sting

**Comment by Ikarrus** [comment]
Added the False Armblade sting to compensate. Read the description for details.

**Comment by Dorsisdwarf** [comment]
oh sweet

**Comment by duncathan** [comment]
:+1:

**Comment by Cheridan** [comment]
cool

**Comment by freerealestate** [comment]
ugh

**Comment by Anonus** [comment]
Time to watch changelings spam the new armblade sting as much as physically possible.

**Comment by Lumipharon** [comment]
I hope you can't double fake blade people, and make them completely helpless.

**Comment by Anonus** [comment]
Hint you can do exactly that

**Comment by Xhuis** [comment]
if someone sees a dude with 2 armblades they'll immediately know that the blades are fake
cmon now

**Comment by Ikarrus** [comment]
I put a ~20 second delay between uses for just that reason @Lumipharon 

If it ends up being OP we can make it longer, but I think that's a lot of time to react to an attacking changeling, or at least call for help.

**Pull request merged**